hack/shellcheck.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -eux
+
+#TODO add make support in https://github.com/openshift/build-machinery-go
+
+if [ "${IS_CONTAINER:-}" != "" ]; then
+  TOP_DIR="${1:-.}"
+  find "${TOP_DIR}" \
+    -path "${TOP_DIR}/vendor" -prune \
+    -o -type f -name '*.sh' -exec shellcheck --format=gcc {} \+
+else
+  docker run --rm \
```

Comment:
Do we want to use `podman` instead of `docker` here?

Reply:
I'd suggest making or using some existing util function to pick whichever is available for portability reasons